packages/hero/index.js

```diff
@@ -65,7 +65,7 @@ function Hero({ data, centered, gaPrefix, className, videoControlsTop }) {
               url={alert.url}
               tag={alert.tag}
               text={alert.text}
-              textColor={backgroundTheme}
+              textColor={backgroundTheme === 'light' ? 'dark' : 'light'}
```

Comment:
2-second hand-wavy idea for the future

💭  We could pass `"invert"` that denotes `backgroundTheme === 'light' ? 'dark' : 'light'` logic
- basically, _the opposite of the current theme_

Reply:
Pass `invert` to `Alert`? I wonder how we would implement that, as that would require `Alert` to be aware of its parent background color 🤔

Reply:
^ inspired by how Vercel handles this at the global level w/ `.invert-theme`
![image](https://user-images.githubusercontent.com/26389321/135308753-e0efd0b0-aa04-4331-b191-7e5db258f83d.png)

Reply:
Gotcha. Yeah I guess we'd need a notion of a proper theme abstraction which we could then "invert." 😅 

Good idea!

Reply:
Probably a farfetched thought — (I'm missing some pieces of how theming is handled)